### PR TITLE
feat: migrating from radix icons to lucide icons

### LINKED ISSUE #7: Double icon sets
Currently we have two icon sets installed 

- https://www.radix-ui.com/icons
- https://lucide.dev/icons/

Seems Shadcn UI by default is using `lucide`, so it's better to follow the same decision.

### PR DESCRIPTION
**Pull request by fandok**
Closes #7 

## Description

Migrating icons from radix to lucide then remove `@radix-ui/react-icons`

## Current Tasks

- [x] Determine how many icons that still imported from radix ui
- [x] Change icons from radix ui to lucide ui by manual comparison
- [x] Remove radix ui react icons package

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ const CSP_ALLOWED_DOMAINS = [
   '*.gstatic.com',
   process.env.NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET,
   process.env.NEXT_PUBLIC_FIREBASE_AUTH_DOMAIN,
+  'https://apis.google.com',
```
Comment (mazipan) [inline next.config.js]: Ini sepertinya udah gak perlu, udah ada wildcard ke *.google.com

**Review by mazipan — COMMENTED** [review]
Tinggal buang unnecessary tambahan di next.config aja

File: next.config.js (hunk 1 above)
Comment (fandok) [reply next.config.js]: updated mas @mazipan

**Review by mazipan — DISMISSED** [review]
Looks 🐐

**Comment by mazipan** [comment]
Build gagal, udah coba rebase dari master terus run `pnpm run build`, sepertinya ada stale code yang masih pakai Radix Icons @fandok 

Screenshot_20231013-184408_Chrome

**Comment by fandok** [comment]
> Build gagal, udah coba rebase dari master terus run `pnpm run build`, sepertinya ada stale code yang masih pakai Radix Icons @fandok
> 
> Screenshot_20231013-184408_Chrome

Updated nih mas @mazipan  udah ganti ke lucide yang masih pake radix icons

**Pull request merged**